README.md

```diff
@@ -10,11 +10,10 @@
 
 ## Compatibility matrix: CRI-O <-> Kubernetes clusters
 
+CRI-O follows the same release cycle and deprecation policy of Kubernetes. For more information visit the [Kubernetes versioning documentation](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md).
```

Comment:
Suggest "policy of Kubernetes" to "policy as Kubernetes".

Reply:
Is "as" expressing possession? I mean, not a native speaker but I want to make sure it's understood that the policy is owned by kubernetes itself

Reply:
No it means they both follow the same release cycle and depraction policy .

Perhaps could make it clear 
`CRI-O and Kubernetes follow the same release cycle and depractation policy.`

Reply:
@runcom, what @redhatdan said, 'as' is not possessive, it's a short way to "the same as" in many instances.

I like Dan's rewording suggestion, just watch the depreciation typo.

Reply:
ok updated